test/functional/specs/C14394.js

```diff
@@ -36,9 +33,10 @@ const apiCalls = ClientFunction(() => {
 
   return window.alloy("event", {
     viewStart: true,
-    // TODO: Change data to match final XDM schema.
     xdm: {
-      key: "value"
+      device: {
+        screenHeight: 1
+      }
```

Comment:
Is this XDM block necessary for any reason? If not, let's just remove it.

Reply:
Event has validation that you must supply either an xdm or data field.

Reply:
It just logs a warning.